app/components/pull-request-page.js

```diff
@@ -302,16 +309,13 @@ export default Ember.Component.extend({
     },
 
     selectJurisdictionFromDropdown(value) {
-      if (value === "__CUSTOM__") {
-        swal({
-          title: "School or Jurisdiction?",
-        })
-      }
       let id = value.split("*")[0]
       let title = value.split("*")[1]
       set(this, "model.standardSet.jurisdiction.id", id)
       set(this, "model.standardSet.jurisdiction.title", title)
-      this.validateThis()
+      if (this.triedToSubmit === true) {
+        this.validateThis()
+      }
```

Comment:
I don't think you need the check here, do you? If the triedToSubmit is guarding all the rendering, I think you can just go ahead and this.validateThis without worrying about the display.